docs/manual/other-topics/upgrade-to-v6.md

```diff
@@ -15,7 +15,7 @@ You should now use [cls-hooked](https://github.com/Jeff-Lewis/cls-hooked) packag
 ```js
 const cls = require("cls-hooked");
 const namespace = cls.createNamespace("....");
-const Sequelize = require("sequelize");
+const Sequelize = require("@sequelize/core");
```

Comment:
```suggestion
const Sequelize = require("sequelize");
```
Since this is related to upgrading to v6 I would not update this one

Reply:
good catch!